docs_user/modules/proc_deploying-backend-services.adoc

```diff
@@ -326,17 +314,17 @@ spec:
       swiftRing:
         ringReplicas: 1
       swiftStorage:
-        replicas: 0
```

Comment:
We need to keep zero replicas here. Swift storage adoption has its own procedure separate from the main OpenStack adoption.